src/CMakeLists.txt

```diff
@@ -489,6 +489,12 @@ if(BUILD_TESTING)
   osh_add_exe(coarsen_test)
   test_func(run_coarsen_test 1 ./coarsen_test)
 
+  if(Omega_h_USE_Gmsh AND Omega_h_USE_Kokkos)
+    osh_add_exe(test_tri_vert_orientation)
+    test_func(run_tri_orientation 1 ./test_tri_vert_orientation ${CMAKE_SOURCE_DIR}/meshes/square4el.msh)
+    set(TEST_EXES ${TEST_EXES} test_tri_vert_orientation)
+  endif()
+
   osh_add_exe(2d_conserve_test)
   test_func(serial_2d_conserve 1 ./2d_conserve_test)
```

Comment:
Given that the test is currently expected to fail, would you please use `will_fail_test_func(...)` as done here:
https://github.com/SCOREC/omega_h/blob/7c2ba19bb3ab2622d20bc4b744e837b0eb35d7d5/src/CMakeLists.txt#L444

Reply:
Thanks for the explanation. I have added the change in the following commit.